### PR TITLE
test pybossa pull request

### PR DESCRIPTION
**Pull request by vasysta**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ To create and preview docs:
 1. Activate virtualenv from PyBossa: `source ../env/bin/activate`.
 2. Build docs with `make html`
 3. Open with your browser of choice `./_build/html/index.html` or open a local webserver in `./_build/html/`
+4. Update
```
Comment (chandra1b) [inline doc/README.md:9]: please add the description correctly

**Comment by chandra1b** [comment]
pl see my comments

**Comment by vasysta** [comment]
I have updated please merge

**Comment by chandra1b** [comment]
OK merged

**Pull request merged**